scout/server/links.py

```diff
@@ -63,6 +63,24 @@ def add_gene_links(gene_obj, build=37):
     gene_obj["cbioportal_link"] = cbioportal_gene(hgnc_symbol)
     gene_obj["civic_link"] = civic_gene(hgnc_symbol)
     gene_obj["iarctp53_link"] = iarctp53(hgnc_symbol)
+    gene_obj["stripy_link"] = stripy_gene(hgnc_symbol)
+    gene_obj["gnomad_str_link"] = gnomad_str_gene(hgnc_symbol)
+
+
+def stripy_gene(hgnc_symbol):
+    link = "https://stripy.org/database/{}"
```

Comment:
This works 👍🏻

<img width="179" alt="image" src="https://user-images.githubusercontent.com/28093618/155958799-641e0b00-2647-4bc4-962f-82f6fe638265.png">